app/queries/get_inactive_applications_from_past_day.rb

```diff
@@ -1,6 +1,7 @@
 class GetInactiveApplicationsFromPastDay
   def self.call(single: true)
     ApplicationForm.current_cycle.joins(:application_choices)
+      .joins(:candidate).where(candidates: { submission_blocked: false, account_locked: false })
```

Comment:
We have this in a few places, should we make a scope on the `Candidate`. Possibly 2 separate scopes, one for transactional emails and one for marketing/nudge emails?

Reply:
Sure!